### ISSUE TITLE
"Read full article" button

### ISSUE THREAD

**Issue by phillipadsmith** [TheTyee/design-article.thetyee.ca#63]
Hey @alexgreen @kidcompassion @MrBryan --

I've been seeing a mobile pattern lately that is to disclose the first paragraph or two of an article, and then to fade it out and show a "Read full article" button.

As much as it's counter-intuitive and thus requires an additional click by the user, I have personally found it to actually make the experience feel less "information overload," i.e., almost like I haven't actually committed to reading the full article yet.

Also, the additional click becomes a very useful data point, e.g., "How many mobile readers actual read past the first paragraph?"

What do you two think? Have you seen what I'm referring to?

**Comment by alexgreen** [comment]
I've seen this before, but can't remember the exact context. I don't recall thinking that it was quite as superfluous as it would be in this case. Can you provide a linked example?
Since the first paragraph is well below the fold on mobile devices I can only think of 1 clear benefit for us: data point (as you mention)

> As much as it's counter-intuitive and thus requires an additional click by the user, I have personally found it to actually make the experience feel less "information overload," i.e., almost like I haven't actually committed to reading the full article yet.

Sounds plausible, but I'm no psychologist so I won't try to generalize. Although I do like the idea that a simple click can add value to the experience.

Basically it prevents people from seeing the actual length of an article right away. Not sure whether withholding this information from the reader is a good idea or not.

**Comment by phillipadsmith** [comment]
I'll try to find some examples. Was looking yesterday, yet couldn't dig one up for some reason -- I think it's something I come across on my phone quite regularly. Let's put it on hold for now and loop back.

**Comment by MrBryan** [comment]
I've seen it a fair bit, too, and although that includes on lots of totally legit and serious sites, for some reason  I associate it more with clickbait sites that want you to see the other ads and capture every conceivable metric while you're on there

So I don't totally think it fits with the Tyee in-depth readers image .  And then we'd have to think about if we would do anything with a metric that would tell us that a percentage of people land on article pages who don't bother clicking past.  My two cents ;)

**Comment by phillipadsmith** [comment]
@MrBryan That's helpful input. True enough about the metric and deciding what to do with that information.

**Comment by alexgreen** [comment]
Yeah, this seems like a solution in search of a problem.

**Comment by phillipadsmith** [comment]
Here are some examples from the NYTimes for future reference:
![img_6569 2](https://cloud.githubusercontent.com/assets/34128/14725095/e40926b0-07ce-11e6-98ea-6d85a25c4390.PNG)
![img_6568 2](https://cloud.githubusercontent.com/assets/34128/14725097/e75dc41a-07ce-11e6-9ae7-9d9c6bc3061e.PNG)
![img_6567 2](https://cloud.githubusercontent.com/assets/34128/14725099/eb136600-07ce-11e6-9783-43e0f1496a5f.PNG)
